website_and_docs/content/blog/2022/selenium-manager-batteries-included-for-selenium-webdriver.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Selenium Manager: batteries included for Selenium WebDriver"
+linkTitle: "Selenium Manager: batteries included for Selenium WebDriver"
+date: 2022-10-20
+tags: ["selenium", "manager", "chromedriver", "geckodriver", "edgedriver"]
+categories: ["releases"]
+author: Boni García ([@boni_gg](https://twitter.com/boni_gg))
+description: >
+   Selenium 4.6.0 is shipped with batteries included, thanks to Selenium Manager.
+---
+
+### Driver management in Selenium WebDriver
+As you probably know, Selenium WebDriver (sometimes called simply _Selenium_, as its umbrella project) is a popular open source library that allows controlling web browsers (e.g., Chrome, Firefox, Edge, etc.) programmatically using different languages (such as Java, JavaScript, Python, Ruby, or C#). Internally, Selenium WebDriver uses the native support implemented by each browser to carry out the automation process. For this reason, we need to place a component called _driver_ (e.g., chromedriver, geckodriver, msedgedriver, etc.) between the script using the Selenium WebDriver API and the browser.
```

Comment:
I would target this post to existing Selenium users who struggle to keep up with driver management, so this feels a little over-explained.